main.js

```diff
@@ -557,20 +545,22 @@ function generateItemDOMFragment(id) {
     let widgetNameLabel = createLabelHTML("name", id, "Name Missing");
     let widgetConfigLabel = createLabelHTML("config", id, "Config missing");
     let widgetsDropDownList = createDropDownHTML(widgetsListName, widgetsList, id, widgetsPlaceholderMessage);
+    let jobsDropDownList = createDropDownHTML(jobsListName, jobsList, id, jobsPlaceholderMessage);
     let configTextField = createConfigTextField(id);
     const fragment = createDOMFragment(
```

Comment:
Not necessary, but using a string template literal would make this much easier to read/edit.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Good suggestion! Thank you